source_test.yaml

```diff
@@ -266,16 +266,3 @@
   key_path: 'OSV'
   repo_username: 'git'
 
-- name: 'test'
-  versions_from_repo: True
-  type: 0
-  ignore_patterns: ['allowlist.json']
-  repo_url: 'blah'
```

Comment:
I'm assuming this is the culprit for a recent spate of

`ValueError: Source repository link must end with /` that just showed up in staging...

Reply:
Yup. Sorry about that 😂

Reply:
Potential cherry-on-top here would be to add some validation so that invalidity doesn't get checked in :-)

Reply:
Just added a pr to merge to this that should be able to fix that :)